code/game/objects/items/devices/propaganda_chip.dm

```diff
@@ -71,7 +71,7 @@ obj/item/device/propaganda_chip/Destroy()
 		if (crew_target_mind)
 			break
 	var/datum/mind/crew_name
-	if (!crew_target_mind)
+	if (!crew_target_mind && !(crew_target_mind.current))
 		crew_name = "Unknown"
 	else
```

Comment:
better be to put it here
`else if(crew_target_mind?.current)`
Cuz if u try to get null.current where `!crew_target_mind` = TRUE, u will get more runtimes